### PR TITLE
Added sliders

### PR DESCRIPTION
**Pull request by haarsh24**

### CODE REVIEW

**Review by xii-vi — APPROVED** [review]
_Everything is in place. You can merge._

File: components/components.html [hunk 1]
```diff
@@ -58,6 +58,9 @@ <h2>
           <div class="list-item">
             <label><a href="#snackbars" class="item-text"> Snackbars</a></label>
           </div>
+          <div class="list-item">
```
Comment (Chiragmodi01) [inline components/components.html:61]: not sure why you've put them in label, but ig it's not necessary.

File: components/components.html [hunk 2]
```diff
@@ -648,7 +651,35 @@ <h3>Our Earth</h3>
...
...520This%2520item%2520already%2520has%2520the%2520label%2520%2522travel%2522.%2520You%2520can%2520add%2520a%2520new%2520label..%250A%2520%2520%2520%2520%253C%252Fp%253E%250A%2520%2520%2520%2520%253Cdiv%253E%250A%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522btn%2520btn-primary%2522%253Eretry%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%2520%253Ci%2520class%253D%2522fa%2520fa-times%2522%2520aria-hidden%253D%2522true%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522snackbar%2520leading%2522%2520id%253D%2522leading%2522%253E%250A%2520%2520%2520%2520%253Cp%2520class%253D%2522snackbar-text%2522%253EYour%2520photo%2520has%2520been%2520archived.%253C%252Fp%253E%250A%2520%2520%2520%2520%253Cbutton%2520class%253D%2522btn%2520btn-primary%2522%253E%2520Undo%253C%252Fbutton%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522fa%2520fa-times%2522%2520aria-hidden%253D%2522true%2522%253E%253C%252Fi%253E%250A%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E%250A%253Cscript%2520src%253D%2522https%253A%252F%252Flitui.netlify.app%252Fjs%252Fsnackbar.js%2522%253E%253C%252Fscript%253E"
```
Comment (Chiragmodi01) [inline components/components.html:656]: link is huge, add it separately!

File: components/components.html [hunk 3]
```diff
@@ -648,7 +651,35 @@ <h3>Our Earth</h3>
...
...on.now.sh/embed?bg=rgba%28134%2C42%2C220%2C1%29&t=night-owl&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=100px&dsblur=100px&wc=true&wa=true&pv=45px&ph=44px&ln=false&fl=1&fm=Hack&fs=13px&lh=153%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522slider-wrapper%2520%2522%253E%250A%2520%2520%253Cinput%2520class%253D%2522simple-slider%2520p2%2522%2520id%253D%2522slider-value%2522%2520type%253D%2522range%2522%2520%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520min%253D%2522100%2522%2520max%253D%2522500%2522%2520value%253D%25221%2522%2520%252F%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522p2%2520h3%2522%253E%250A%2520%2520%253Cspan%2520class%253D%2522btn%2520btn-secondary%2522%253E100%253C%252Fspan%253E%250A%2520%2520%253Cspan%2520id%253D%2522value%2522%2520class%253D%2522btn%2520btn-secondary%2522%253E500%253C%252Fspan%253E%250A%253C%252Fdiv%253E%250A%250A%252F**%2520JS%2520Script%2520**%252F%250A%253Cscript%253E%250Adocument.getElementById%28%2522slider-value%2522%29.oninput%2520%253D%2520function%2520%28%29%2520%257B%250A%2520%2520document.getElementById%28%2522value%2522%29.innerHTML%2520%253D%2520this.value%253B%250A%257D%253B%250A%253C%252Fscript%253E%250A"
```
Comment (Chiragmodi01) [inline components/components.html:678]: same for all carbon links

File: components/components.html [hunk 4]
```diff
@@ -664,6 +695,7 @@ <h2>
   </footer>
   </div>
 </body>
-<script src="snackbar.js"></script>
+<script src="../js/snackbar.js"></script>
```
Comment (Chiragmodi01) [inline components/components.html:698]: two same script tags??

**Review by Chiragmodi01 — APPROVED** [review]
added some sanity comments.
overall looks good to merge

File: components/components.html (hunk 4 above)
Comment (haarsh24) [reply components/components.html:698]: By mistake , will remove this

**Pull request merged**